src/main/java/picard/nio/GoogleStorageUtils.java

```diff
@@ -0,0 +1,46 @@
+package picard.nio;
+
+import com.google.cloud.http.HttpTransportOptions;
+import com.google.cloud.storage.StorageOptions;
+import com.google.cloud.storage.contrib.nio.CloudStorageConfiguration;
+import com.google.cloud.storage.contrib.nio.CloudStorageFileSystemProvider;
+import shaded.cloud_nio.com.google.api.gax.retrying.RetrySettings;
+import shaded.cloud_nio.org.threeten.bp.Duration;
+
+
+/**
+ * Created by farjoun on 11/13/17.
```

Comment:
Needs a better description.  Include the decision to include this so that we can have an optional dependency on GCS.